Gemfile.lock

```diff
@@ -192,10 +192,11 @@ GEM
       pry (>= 0.13, < 0.15)
     public_suffix (5.0.1)
     racc (1.6.2)
-    rack (2.2.6.4)
+    rack (3.0.7)
```

Comment:
Are there any concerns about moving a major version of rack too? Are we on v2 most other places?

Reply:
There's no web app, so shouldn't be a problem.